foundation/foundation-configuration-store/src/main/java/com/boozallen/aissemble/configuration/store/ConfigLoader.java

```diff
@@ -82,15 +89,15 @@ public Set<Property> loadConfigs(String baseURI, String environmentURI) {
      * @return Set of properties.
      */
     public Set<Property> loadConfigs(String baseURI) {
-        return loadURI(baseURI);
+        return loadPropertyURI(baseURI);
     }
 
     /**
      * Inspects config file(s) at the given URI and gathers all properties.
      * @param URI Location housing configuration yamls.
      * @return Set of Property objects.
      */
-    private Set<Property> loadURI(String URI) {
+    private Set<Property> loadPropertyURI(String URI) {
```

Comment:
Q: `loadPropertyURI` sounds to me like it implies that only one property is loaded. Maybe we could call this something like `loadConfigsURI` ?

Reply:
How do you feel about `loadPropertiesURI`. Was trying to avoid using config since it sort of ambiguous whether it would be loading properties or policies.